framework/src/modules/chain/synchronizer/block_synchronization_mechanism.js

```diff
@@ -52,7 +54,40 @@ class BlockSynchronizationMechanism {
 		// TODO: handle bestPeer and move on to step 2 defined in
 		// https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md#block-synchronization-mechanism
 		// ...
-		this.active = false;
+
+		// The node requests the last common block C from P (Peer).
+		const { data: networkLastBlock } = await this.channel.invoke(
```

Comment:
if there is network failure, it will keep `this.active` to be true

Reply:
Reverted this change.

Reply:
I think if this `invoke` fails `this.active` is still true?

Reply:
Moved the call to the try block